src/bindings/config.ts

```diff
@@ -60,7 +60,7 @@ export const loadConfig = async (context: Context): Promise<BotConfig> => {
     payout: {
       networkId: networkId,
       rpc: rpc,
-      privateKey: privateKey,
+      privateKeyEncrypted: privateKeyEncrypted || null,
```

Comment:
`privateKeyEncrypted` is already `string | undefined`, so I think converting it to `string | null` is unnecessary step, but not a big deal.
It would also simplify `Type.Union([Type.Null(), Type.String()])` into `Type.Optional(Type.String())`

Reply:
`null` is more expressive that it is intentionally set to a void value instead of some type of mistake happening.